starter-code/src/viking.js

```diff
@@ -1,8 +1,40 @@
 // Soldier
-function Soldier() {}
+function Soldier(health,strength){
+    this.health = health;
+    this.strength = strength;
+  
+  }
 
+  Soldier.prototype.attack = function(){
+    return this.strength;
+  }
+  Soldier.prototype.receiveDamage = function(Damage){
+  this.health = this.health - Damage;
+  }
+  
+    
 // Viking
-function Viking() {}
+function Viking(name,health,strength) {
+    Soldier.call(this, name, owner);
+    this.name = name;
+};
+
+function attack(){
+    return Viking.strength;
+}
+function battleCry(){
+
+}
+ Viking.prototype.receiveDamage = function(Damage){
```

Comment:
No nombréis variables con la primera en mayúscula, es decir, en vez de `Damage`, poned `damage`. Las mayúsculas se dejan para definir la función constructora ;)